### PR TITLE
[v6r15] Bug fixes

### PR DESCRIPTION
**Pull request by fstagni**
Bad merge fixed, plus few minor bug fixes.

### CODE REVIEW

File: Resources/Storage/test/test_RFIOPlugIn.py [hunk 1]
```diff
@@ -1,7 +1,7 @@
 """ test for RFIO plugin
```
Comment (atsareg) [inline Resources/Storage/test/test_RFIOPlugIn.py]: Pleeeeease, do not rename to a lower t ! Let's make a convention to start non valid yet tests with No_Test_<...>

Comment (fstagni) [inline Resources/Storage/test/test_RFIOPlugIn.py]: I renamed in "FIXME_Test_<>"

**Comment by atsareg** [comment]
Rename also Test_FTSJOB ?

**Comment by fstagni** [comment]
Done.

**Comment by atsareg** [comment]
Review OK

**Comment by fstagni** [comment]
I made several fixes that deserve a new review.

File: tests/Jenkins/dirac-cfg-update.py [hunk 2]
```diff
@@ -34,7 +40,16 @@
...
 else:
-  localConfigFile = './etc/dirac.cfg'
+  print "WORKSPACE: %s" % os.path.expandvars('$WORKSPACE')
+  if os.path.isfile( os.path.expandvars('$WORKSPACE')+'/PilotInstallDIR/etc/dirac.cfg' ):
+    localConfigFile = os.path.expandvars('$WORKSPACE')+'/PilotInstallDIR/etc/dirac.cfg'
+  elif os.path.isfile( os.path.expandvars('$WORKSPACE')+'/ServerInstallDIR/etc/dirac.cfg' ):
+    localConfigFile = os.path.expandvars('$WORKSPACE')+'/ServerInstallDIR/etc/dirac.cfg'
+  elif os.path.isfile( './etc/dirac.cfg' ):
+    localConfigFile = './etc/dirac.cfg'
+  else:
```
Comment (anusharanganathan) [inline tests/Jenkins/dirac-cfg-update.py]: This check of a series of paths in the configuration in order is done in multiple places. Maybe it is useful to create a utility to do this for everyone?

Comment (rajanandakumar) [inline tests/Jenkins/dirac-cfg-update.py]: Apologies - that was me (Raja) commenting above under a different identity by mistake :)

Comment (fstagni) [inline tests/Jenkins/dirac-cfg-update.py]: "multiple identities" Raja... suspicious! ;-)

Anyway, on the comment: this is for test purposes, no need for further extension, IMHO an utility here would not make things simpler I am afraid.

**Pull request merged**